tests/automation_framework/data/worker/worker_setstatus.ini

```diff
@@ -0,0 +1,30 @@
+[DEFAULT]
+method=WorkerSetStatus
+id=3
+params:{"workerId":"",
+        "status": 1}
+
+[18269]
+workerId=4d465977454159484b6f5a497a6a3043415159464b34454541416f4451674145
```

Comment:
why are we hard coding the workerId? Shouldn't this be filled automatically by worker retrieve params file?

Also why do you need to remove the status parameter when the test case is just for unknown parameter